proto/buf/alpha/registry/v1alpha1/webhook.proto

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+// WebhookService exposes the functionality for a caller to
+// subscribe to or unsubscribe to a Webhook for a given repository.
+service WebhookService {
+  rpc SubscribeToRepository(SubscribeToRepositoryRequest) returns (SubscribeToRepositoryResponse) {}
+  rpc UnsubscribeToRepository(UnsubscribeToRepositoryRequest) returns (UnsubscribeToRepositoryResponse) {}
+  rpc TestRepository(TestRepositoryRequest) returns (TestRepositoryResponse) {}
```

Comment:
What is this for?

Reply:
The intention here is to provide a way to initiate an event via curl to test out a registered webhook end to end

We can provide this as a way for the user to test out their webhook without doing a bunch of pushes but if we're not convinced we need this and if we want to avoid a backwards incompatible change, I can remove it!

Reply:
Let's remove it now then!

Reply:
I can hack a local endpoint to do it instead of using this generated method. We can add something like this on short order if we think it'll be useful